src/main/java/org/jvnet/hudson/test/FakeLauncher.java

```diff
@@ -10,11 +10,11 @@
  * Fake a process launch.
  *
  * @author Kohsuke Kawaguchi
- * @see PretendSlave
+ * @see PretendAgent
```

Comment:
There is no Java class named `PretendSlave`, so this change causes javadoc generation to fail.  Refer to the [Pipeline overview](https://ci.jenkins.io/job/Core/job/jenkins-test-harness/job/PR-1019/1/pipeline-overview/) for the other failures.
```suggestion
 * @see PretendSlave
```